server/config/persist_options.go

```diff
@@ -30,6 +32,8 @@ import (
 // PersistOptions wraps all configurations that need to persist to storage and
 // allows to access them safely.
 type PersistOptions struct {
+	// configuration -> ttl value
+	ttl             map[string]*cache.TTLString
```

Comment:
What if the PD restart?

Reply:
As we didn't persist it, so we need lightning and BR to call this API with a short interval (like once a minute)